release-notes/unreleased/IGAPP-1005-double-separation-line-poi-list-bas.yml

```
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1005
+show_in_stores: true
+platforms:
+  - web
+en: Removed double separation line in location list view mobile.
+de: Doppelte Trennlinie in der mobilen Listenansicht der Lokationen entfernt.
```

Lokationen is definitely not a term we use:
```suggestion
en: Removed double separation line in location list view mobile.
de: Doppelte Trennlinie in der mobilen Listenansicht der Orte entfernt.
```